addon/plugins/article-structure-plugin/utils/structure.ts

```diff
@@ -38,26 +45,24 @@ export function constructStructureNodeSpec(config: {
     },
     allowSplitByTable: config.allowSplitByTable,
     toDOM(node) {
-      return [
-        'div',
-        {
-          property: node.attrs.property as string,
-          typeof: node.attrs.typeof as string,
-          resource: node.attrs.resource as string,
+      return renderRdfaAware({
+        renderable: node,
+        tag: 'div',
+        attrs: {
+          ...node.attrs,
+          class: 'say-editable',
         },
-        0,
-      ];
+        content: 0,
+      });
     },
     parseDOM: [
       {
         tag: 'div',
+        priority: 55,
         getAttrs(element: HTMLElement) {
-          if (
-            hasRDFaAttribute(element, 'property', SAY('hasPart')) &&
-            hasRDFaAttribute(element, 'typeof', type) &&
-            element.getAttribute('resource')
-          ) {
-            return { resource: element.getAttribute('resource') };
+          const resource = element.getAttribute('resource');
+          if (hasRDFaAttribute(element, 'typeof', type) && resource) {
```

Comment:
I suppose we could do this check on the properties attribute of the incoming html-element now? 
e.g.:
```js
attrs.properties.some((prop) => prop.predicate === 'typeof' && prop.object.resource === type)
```

Reply:
I'm not sure what you mean. Do you mean to use `getRdfaAttrs()` to process the element and use the `properties` member of that attrs object?